pages/user/_id.vue

```diff
@@ -141,6 +161,15 @@ export default {
         sum += mod.downloads
       }
 
+      return this.formatNumber(sum)
+    },
+    sumFollows() {
```

Comment:
I know `sumDownloads` was already a method, but I would either make `sumDownloads` & `sumFollows` a computed property or a data property. It doesn't *really* matter if you don't want to change it.